src/App.js

```diff
@@ -4,19 +4,21 @@ import './App.css';
 import todos from './api/todos';
 import users from './api/users';
 
+import { TodoList } from './components/TodoList';
+
+const compiledList = (todosList, usersList) => (
```

Comment:
```suggestion
const getTodosWithUsers = (todosList, usersList) => (
```
функция должна называться как глагол, что-то сделать